src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java

```diff
@@ -125,4 +126,32 @@ Integer getInt(Map<Option, ?> options) {
   List<Boolean> testPermissions(String projectId, List<String> permissions);
 
   // TODO(ajaykannan): implement "Organization" functionality when available (issue #319)
+
+  /**
+   * Creates the new lien which applies to the resource denoted by the parent field.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  Lien createLien(Lien lien);
+
+  /**
+   * Deletes the Lien by `name`.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  void deleteLien(String name);
+
+  /**
+   * Gets the Lien by `name`.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  Lien getLien(String name);
+
+  /**
+   * Lists all the Liens applied to the parent resource.
+   *
+   * @throws ResourceManagerException upon failure
+   */
+  Tuple<String, Iterable<Lien>> listLiens(String parent, Map<Option, ?> options);
```

Comment:
This is a bad idea. See go/nopair
We should never have created this class, and please don't add new usages. We need a semanic carrying class here instead

Reply:
I had follow the same behaviour as [Project->list](https://github.com/googleapis/java-resourcemanager/blob/master/src/main/java/com/google/cloud/resourcemanager/spi/v1beta1/ResourceManagerRpc.java#L86). I think same pattern follow in other google client as well for example: [bigQuery](https://github.com/googleapis/java-bigquery/blob/master/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/BigQueryRpc.java#L154).

Reply:
Bad decisions in other APIs are not a justification for copying them here. It was a mistake to create the Tuple class in the first place. No new usages of it, please.

Reply:
Correct I have already removed tuple and add semantic carrying class instead of that.